### PR TITLE
fix timewheel error

### PR DESCRIPTION
**Pull request by georgehao**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: time/timer.go [hunk 1]
```diff
@@ -507,6 +506,7 @@ func (w *TimerWheel) AddTimer(f TimerFunc, typ TimerType, period time.Duration,
 		return nil, ErrTimeChannelClosed
 	}
 
+	println("add timer")
```
Comment (AlexStocks) [inline time/timer.go:509]: pls delete this line

**Comment by wongoo** [comment]
@georgehao can u tell what this pr fix?

File: time/timer.go (hunk 1 above)
Comment (georgehao) [reply time/timer.go]: done

**Comment by georgehao** [comment]
> @georgehao can u tell what this pr fix?

**Comment by georgehao** [comment]
> @georgehao can u tell what this pr fix?

**Comment by georgehao** [comment]
> @georgehao can u tell what this pr fix?

just fix bug for : 

when the timer node in timewheel is 0, the timewheel run goroutine exit. 

So getty server start, find no client connect, so the timewheel run goroutine exit,  but the timewheel is still be used. so when the client connect to the server, this will call `addtimer` to the timewheel .

When the timewheel's channel full, client can't connect to the server

File: time/timer.go [hunk 2]
```diff
@@ -200,46 +202,42 @@ func NewTimerWheel() *TimerWheel {
 			if !w.enable.Load() {
```
Comment (pantianying) [inline time/timer.go:202]: It seems that the new code 'enable' must be true, so should we consider removing this code?

Comment (AlexStocks) [reply time/timer.go:202]: @pantianying no. pls check the enable in timer.Stop() func.

**Review by AlexStocks — APPROVED** [review]

**Review by zhaoyunxing92 — APPROVED** [review]
approved these changes

**Review by LaurenceLiZhixin — APPROVED** [review]

**Review by xiaoliu10 — APPROVED** [review]

**Pull request merged**